torch/csrc/jit/script/parser.cpp

```diff
@@ -450,11 +450,24 @@ struct ParserImpl {
       // There is an assignment operator, parse the RHS and generate the
       // assignment.
       auto rhs = parseExpOrExpTuple();
-      L.expect(TK_NEWLINE);
       if (maybeOp.value()->kind() == '=') {
+        std::vector<Expr> lhs_list = {lhs};
+        while (L.nextIf('=')) {
+          lhs_list.push_back(rhs);
+          rhs = parseExpOrExpTuple();
+        }
+        if (type.present()) {
+          throw ErrorReport(type.range())
+              << "Annotated multiple assignment is not well-defined in python";
```

Comment:
say `not supported` instead of `not well-defined`

Reply:
Sure, this is what i'm referencing fyi https://mypy.readthedocs.io/en/latest/type_inference_and_annotations.html